cpmonitor/migrations/0028_merge_20230827_1442.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 4.1.7 on 2023-08-27 12:42
```

Comment:
To keep the number of migrations as low as possible, it would be better to rebase your branch and recreate the original migration.

Reply:
Done